app/views/order_items/index.html.erb

```diff
@@ -1,32 +1,43 @@
-<h1>Your Cart</h1>
+<div class="panel panel-warning">
+  <div class="panel-body">
+    <h2>Your Cart  <span class="totalcart">Total:<%= @order.order_total%></span>
+</h2>
+    <!-- <% if @order.nil? %>
+      <p>No items in cart. </p>
+    <% else %> -->
+    <ul>
+      <% @order_items.each do |item| %>
+        <% product = Product.find(item.product_id) %>
+        <li>
+          <%= image_tag product.image, height: '100', width: '100', class: "productpic" %>
```

Comment:
Who are you,  why am I receiving emails from you, and did I sign up to
receive them? If so,  when? Respond today. Thank you.

Thomas Murphy
On Jun 13, 2016 3:56 PM, "Jennie Buechner" <notifications@github.com> wrote:

> In app/views/order_items/index.html.erb
> <https://github.com/vconklin/betsy/pull/1#discussion_r66883791>:
>
> > @@ -1,32 +1,43 @@
> > -<h1>Your Cart</h1>
> > +<div class="panel panel-warning">
> > +  <div class="panel-body">
> > +    <h2>Your Cart  <span class="totalcart">Total:<%= @order.order_total%></span>
> > +</h2>
> > +    <!-- <% if @order.nil? %>
> > +      <p>No items in cart. </p>
> > +    <% else %> -->
> > +    <ul>
> > +      <% @order_items.each do |item| %>
> > +        <% product = Product.find(item.product_id) %>
> > +        <li>
> > +          <%= image_tag product.image, height: '100', width: '100', class: "productpic" %>
>
> Is this image important to the understanding of what is on the page? If
> so, you should include an alt attribute for screen readers/accessibility.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/vconklin/betsy/pull/1/files/d404571ad98034b0f9de6d01b688165e61307f78#r66883791>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAZjcQeFeTyW0oR-00DhzP3DNL6NTrc5ks5qLeAzgaJpZM4Ik99H>
> .
>